backend/clubs/models.py

```diff
@@ -126,6 +126,12 @@ def get_event_small_file_name(instance, fname):
     )
 
 
+def get_membership_image_file_name(instance, fname):
+    return os.path.join(
+        "membership", "{}.{}".format(instance.code, fname.rsplit(".", 1)[-1])
```

Comment:
The instance here is an instance of `Membership`, right? Because `Membership` has no field `code` field I think that this will default to `None` and you could get conflicts if two users upload files with the same filename.

Maybe you can do something like
```
"{}.{}.{}".format(instance.club.code, instance.person.username, fname.rsplit(".", 1)[-1])
```
which will be distinct because there is only one membership for each (person, club) pairing

Reply:
Yup totally. I missed that when I copied over from the other similar methods.